keras/utils/conv_utils_test.py

```diff
@@ -66,16 +66,34 @@ def test_convert_data_format(self):
       conv_utils.convert_data_format('invalid', 2)
 
   def test_normalize_tuple(self):
-    self.assertEqual((2, 2, 2),
-                     conv_utils.normalize_tuple(2, n=3, name='strides'))
-    self.assertEqual((2, 1, 2),
-                     conv_utils.normalize_tuple((2, 1, 2), n=3, name='strides'))
-
-    with self.assertRaises(ValueError):
-      conv_utils.normalize_tuple((2, 1), n=3, name='strides')
-
-    with self.assertRaises(ValueError):
-      conv_utils.normalize_tuple(None, n=3, name='strides')
+    self.assertEqual((2, 2, 2), conv_utils.normalize_tuple(
+        2, n=3, name='strides', allow_zero=True))
+    self.assertEqual((2, 1, 2), conv_utils.normalize_tuple(
+        (2, 1, 2), n=3, name='strides', allow_zero=True))
+    self.assertEqual((1, 2, 3,), conv_utils.normalize_tuple(
+        (1, 2, 3), n=3, name='pool_size'))
+    self.assertEqual((3, 3, 3), conv_utils.normalize_tuple(
+        3, n=3, name='pool_size'))
+
+    with self.assertRaises(ValueError) as ctx:
```

Comment:
Please use assertRaisesRegex to check the error type and message at same time.

Reply:
Done